packages/api-gateway-service/src/helpers.ts

```diff
@@ -0,0 +1,15 @@
+const fetch = require( 'node-fetch' );
+
+/*This consists of the list of microservices which needs to be part of the gateway*/
+export const serviceList = [
+  { name: 'User Group', url: process.env.USER_GROUP }
+];
+
+/* Function to fetch the public key from internal IDP*/
+// export const publicKey = () => {
+//   return fetch( process.env.INTERNAL_IDP )
+//     .then( ( res: any ) => res.json() )
+//     .then( ( res: any ) => {
+//       return res.public_key;
+//     } );
+// };
```

Comment:
Remove it for now. Add it back when it's implemented.

Reply:
removed this code